framework/widgets/ActiveField.php

```diff
@@ -315,10 +317,17 @@ public function error($options = [])
      *
      * @return $this the field object itself
      */
-    public function hint($content, $options = [])
+    public function hint($content = null, $options = [])
```

Comment:
In order for it to be merged in 2.0 it should not change public method signature.